e2e/kots-release-install-v3/config.yaml

```diff
@@ -0,0 +1,148 @@
+apiVersion: kots.io/v1beta1
```

Comment:
we can perhaps follow this up in a different story, but i think it'd be better if the config was actually used in the app. meaning that these config options are actually templated into `kind: HelmChart`